web-registry/src/lib/ledger.ts

```diff
@@ -1,7 +1,7 @@
 import getApiUri from './apiUri';
 
 export const chainId = process.env.REACT_APP_LEDGER_CHAIN_ID;
-export const nctBasket = process.env.REACT_APP_NCT_BASKET;
+export const nctBasket = Boolean(process.env.REACT_APP_NCT_BASKET);
```

Comment:
could you add a note here about what value/s this should be set to turn on/off the basket feature? because unless i'm misunderstanding something you might end up with a scenario like this:

you want the feature turned off:
```
REACT_APP_NCT_BASKET=false
```

but the boolean conversion evaluates to true (even in the case of a 'false' string value):
```
console.log(Boolean(process.env.REACT_APP_NCT_BASKET)) => true
```

i'm working on the assumption that REACT_APP env var's don't have a special boolean value parser though, and that these values just get read in as strings.

Reply:
You're right this might be confusing. Since REACT_APP_VAR are sent as string, we've been checking for any non-empty string as a valid flag so far.
Will change the flag evaluation to `export const nctBasket = process.env.REACT_APP_NCT_BASKET === 'true';` to make it more explicit and avoid the `Boolean('false')`akward case.